### ISSUE TITLE
[Bug]: Fix Accessibility Issue - CTRL+Click on links should open new tabs

### ISSUE THREAD

**Issue by andrewballantyne** [opendatahub-io/odh-dashboard#527]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When you ctrl+click on the KFNBC `Jupyter` application and it doesn't open a new tab, we violated accessibility.

### Expected Behavior

We should fix the KFNBC link -- I do not believe we have others but those should be also link-enabled. 

Using `<Button>`s to do everything can be problematic for accessibility. We need to make sure PF is giving us an anchor under the hood so we are compliant with the web standard.

### Steps To Reproduce

1. Have KFNBC enabled
2. CTRL+Click on the launch link

### Open Data Hub Version

```yml
2.1
```
